pandas/tests/window/test_apply.py

```diff
@@ -138,3 +138,31 @@ def test_invalid_kwargs_nopython():
         Series(range(1)).rolling(1).apply(
             lambda x: x, kwargs={"a": 1}, engine="numba", raw=True
         )
+
+
+def test_rolling_apply_args_kwargs():
+    # GH 33433
+    def foo(x, par):
+        return np.sum(x + par)
+
+    df = DataFrame({"gr": [1, 1], "a": [1, 2]})
+
+    idx = Index(["gr", "a"])
+    expected = DataFrame([[11.0, 11.0], [11.0, 12.0]], columns=idx)
+
+    result = df.rolling(1).apply(foo, kwargs={"par": 10})
```

Comment:
Could you parameterize over the kwargs & args?

`pytest.mark.parameterize('args_kwargs', [[None, {"par": 10}], [(10,), None]])`